<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitializeWindow</class>
 <widget class="QMainWindow" name="InitializeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>InitializeWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/LOGO-AT.ico</normaloff>../res/LOGO-AT.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="styleSheet">
       <string notr="true">color:#DDDDDD;
background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 #000000, stop:0.7 #474e5e);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>24</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <property name="bottomMargin">
        <number>24</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">color:#DDDDDD;background: #272e3e;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>24</number>
          </property>
          <property name="topMargin">
           <number>24</number>
          </property>
          <property name="rightMargin">
           <number>24</number>
          </property>
          <property name="bottomMargin">
           <number>24</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="5" column="0" colspan="2">
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#widget_3{background: qlineargradient(x1:0,y1:1,x2:1,y2:1,stop:0.1 #f4716e, stop:0.5 #42f875, stop:0.9 #1edffc);}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_status">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background: qlineargradient(x1:1,y1:1,x2:1,y2:0,stop:0 transparent, stop:0.2 #272e3e);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_line_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>2</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>2</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: qlineargradient(x1:0,y1:1,x2:1,y2:1,stop:0 transparent, stop:0.5 #1edffc, stop:1 transparent);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QLabel" name="label_info">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_icon_text">
            <property name="text">
             <string>@ GNU GENERAL PUBLIC LICENSE</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_icon">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{border-image:url(:/img/res/LOGO-AT.png) 4 4 4 4 stretch stretch;background-color:transparent}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_line_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>2</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>2</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: qlineargradient(x1:0,y1:1,x2:1,y2:1,stop:0 transparent, stop:0.5 #1edffc, stop:1 transparent);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_logo">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{border-image:url(:/img/res/LOGO-ATWR.png) 4 4 4 4 stretch stretch;background-color:transparent}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actioncalib">
   <property name="text">
    <string>calib</string>
   </property>
  </action>
  <action name="actionparam">
   <property name="text">
    <string>param</string>
   </property>
  </action>
  <action name="actionconfig">
   <property name="text">
    <string>config</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
